tapdb/migrations.go

```diff
@@ -173,7 +173,7 @@ func newReplacerFile(parent fs.File, replaces map[string]string) (*replacerFile,
 
 	contentStr := string(content)
 	for from, to := range replaces {
-		contentStr = strings.Replace(contentStr, from, to, -1)
```

Comment:
nit: seems unrelated to the PR at hand?

Reply:
Yes, though it also causes a postgres test failure, which is surprising.

Reply:
Help me understand why the test failure, post function change, is surprising

Reply:
Should be no behavior change moving from `Replace` with a 3rd arg of `-1` to `ReplaceAll`, according to the `strings` package documentation. But one of the unit tests running under postgres failed on an SQL issue. Which I don't think was a known flake.

Will flakehunt locally.